composer.json

```diff
@@ -14,8 +15,8 @@
         }
     ],
     "require": {
-        "php": "^7.0",
-        "phpspec/phpspec": "6.*"
+        "php": "^7.2",
```

Comment:
I changed this because `phpspec` v6 works only with php >= 7.2: https://packagist.org/packages/phpspec/phpspec#6.0.0